src/network/ergo/wallets/SafewWallet.tsx

```diff
@@ -0,0 +1,31 @@
+import { ErgoBox, ergoBoxFromProxy } from '@ergolabs/ergo-sdk';
+import React from 'react';
+import { catchError, from, map, Observable, of, tap, throwError } from 'rxjs';
+
+import { ReactComponent as SafewLogo } from '../../../assets/icons/safew-logo-icon.svg';
+import { Wallet } from '../../common';
+
+const connectWallet = (): Observable<any> => {
+  if (!ergoConnector?.safew) {
+    return throwError(() => new Error('EXTENSION_NOT_FOUND'));
+  }
+  return from(ergoConnector.safew.connect()).pipe(
+    tap(() => (window.safew = Object.freeze(new SafewErgoApi()))),
+    catchError(() => of(true)),
+  );
+};
+
+const getUtxos = (): Observable<ErgoBox[]> =>
+  from(window.safew.get_utxos()).pipe(
+    map((bs) => bs?.map((b) => ergoBoxFromProxy(b))),
+    map((data) => data ?? []),
+  );
+
+export const SafewWallet: Wallet = {
+  name: 'Safew',
+  icon: <SafewLogo />,
+  experimental: true,
+  extensionLink: 'https://github.com/ThierryM1212/SAFEW/releases',
```

Comment:
It has to be a link to the chrome store.

Reply:
Thanks
Google only accept manifest v3 new extension since 17th january, and https://bugs.chromium.org/p/chromium/issues/detail?id=1173354 prevent to make any extension using a wasm library...

Opera store could be good enough ?

Reply:
Yes. The button should be available only in the Opera browser.